api/app/Http/Controllers/BaseController.php

```diff
@@ -153,68 +157,4 @@ public function deleteMany(Request $request)
 
         return response(null, 204);
     }
```

Comment:
@Redjik @xiphiaz Using @model->set<Something>() for relations sounds good to me as well. I'll update the user creation accordingly with the credentials relation to adopt this.